core/src/main/java/com/sequenceiq/cloudbreak/service/identitymapping/AwsIdentityMappingService.java

```diff
@@ -0,0 +1,95 @@
+package com.sequenceiq.cloudbreak.service.identitymapping;
+
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cloud.IdentityService;
+import com.sequenceiq.cloudbreak.cloud.init.CloudPlatformConnectors;
+import com.sequenceiq.cloudbreak.cloud.model.Platform;
+import com.sequenceiq.cloudbreak.cloud.model.Variant;
+import com.sequenceiq.cloudbreak.converter.spi.CredentialToCloudCredentialConverter;
+import com.sequenceiq.cloudbreak.dto.credential.Credential;
+
+@Component
+public class AwsIdentityMappingService {
```

Comment:
Given that this service always produces fake mappings, maybe a name like `AwsMockIdentityMappingService` would be more appropriate.

Reply:
renamed